.travis.yml

```diff
@@ -0,0 +1,19 @@
+dist: xenial
+sudo: required
+language: python
+
+python:
+- 3.6
+
+before_install:
+- bash <(wget -q -O- http://neuro.debian.net/_files/neurodebian-travis.sh)
+- sudo apt-get -q install -y git-annex-standalone
+
+install:
+- pip install datalad
```

Comment:
Does it make a difference if you specify `pip3` instead of `pip`? I am not sure if it is irrelevant to say pip3 given that you are specifying Python 3.6 above. Just wondering...

Reply:
It shouldn't make any difference when Python 3 is used.

Reply:
I just tested and it seems like it doesn't make a difference with `pip3`: https://travis-ci.com/joeyzhou98/conp-dataset/jobs/218328117 compared to a previous build with only `pip`

Reply:
Super, thanks!!